compiler/z/codegen/OMRMemoryReference.cpp

```diff
@@ -1611,7 +1611,7 @@ bool OMR::Z::MemoryReference::tryBaseIndexDispl(TR::CodeGenerator* cg, TR::Node*
    bool debug = false;
    TR::Register* breg = NULL;
    TR::Register* ireg= NULL;
-   static int folded = 0;
+   int folded = 0;
```

Comment:
This is only used as part of a print below. Perhaps we can remove this field altogether if it's no longer relevant. It will not yield the correct information if it's used for JITServer diagnostics if it's left `static`. And it may not yield useful information if it's not `static` for regular JIT Compiles. So I'm not sure what the best solution is here.

Reply:
Let's remove this variable.

Reply:
Fixed here: fa79768